source/JsMaterialX/JsMaterialXCore/JsTypes.js

```diff
@@ -1,26 +1,15 @@
 // jsTypes
-addWrapper(function(Module, api) {
+addWrapper(function(Module, api = {}) {
+    Object.keys
+    api = {
+        ...api,
+        ...Module
```

Comment:
This copies the entire Emscripten Module into this api object, which will most likely mess up the wrapping. I think @sdunkel might already have a way to deal with constants.

Reply:
Well, getting rid of the wrapper altogether is the solution, meanwhile we can just minimise it. And it won't mess up anything because wrapper functions are afterwards. This whole JS thing is weird because it is re-exporting things, so I don't believe there is anything inside the Module at the moment that should not be exposed

Reply:
Ah, I see, I understand now. I thought this was aggregated somewhere, not that it was accumulating. Thanks for the explanation